tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -145,7 +172,7 @@ Create table
 
 Validate results table column headings
     [Tags]  HappyPath
-    user checks table column heading contains  1   1   2020 Week 14
+    user checks table column heading contains  1   1  Female
```

Comment:
We should add more checks to the table itself, to check the data is correct - and cross reference what is being produced with the CSV file to make sure it is actually correct.